sheetsync/sheetsync/main.py

```diff
@@ -364,24 +364,25 @@ def sync_playlists(self, rows):
 		overwriting the entire playlists table"""
 		playlists = []
 		for row in rows:
-			if len(row) != 3:
+			if len(row) != 5:
 				continue
-			tags, _, playlist_id = row
+			tags, _, name, playlist_id, show_in_desctiption = row
```

Comment:
This typograghical error could be a problem later.

Reply:
Fixed.